ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Ledger/Query.hs

```diff
@@ -365,18 +392,20 @@ decodeQueryAnytimeResult :: QueryAnytime result -> forall s. Decoder s result
 decodeQueryAnytimeResult GetEraStart = decode
 
 encodeQueryHardForkResult ::
-     SListI xs
+     (ToCBOR (LedgerConfig (HardForkBlock xs)), SListI xs)
   => QueryHardFork xs result -> result -> Encoding
 encodeQueryHardForkResult = \case
     GetInterpreter -> encode
```

Comment:
I opened Issue #3013 to come back and replace these uses of `encode`/`decode` with `toCBOR`/`fromCBOR`.